artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/shared/components/login/login.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="self-center flex space-x-2">
+    @if(userData().isAuthorized) {
+    <p>{{userData().userName}}</p>
+    <img class="rounded-full max-h-10" src="{{userData().userPhotoUrl}}"
```

Comment:
nigdy nie rozumiałem czemu ludzie nie robią po prostu [src]="userData().userPhotoUrl" a teraz widze że to daje intellisense też ale still musiałem zwrócić uwagę xD

Reply:
za moich czasow